impl/src/main/java/org/ehcache/impl/persistence/DefaultDiskResourceService.java

```diff
@@ -113,7 +121,7 @@ public void releasePersistenceSpaceIdentifier(PersistenceSpaceIdentifier<?> iden
       }
     }
     if (name == null) {
-      throw new CachePersistenceException("Unknown space " + identifier);
+      throw newCachePersistenceException(identifier);
```

Comment:
The condition on which this exception is thrown is the same everywhere (no value matching the identifier is found in the map).  Would it be possible to push the throwing of this exception in a single method deeper in the call stack?